### PR TITLE
docs: ✏️ update pull request template

### LINKED ISSUE #53: Update Pull Request template.

### PR DESCRIPTION
**Pull request by CristiPV**
### This pull request solves issue:

Closes: #53

### Which app is this issue related to?

- [x] REST API
- [ ] E-Learning React App
- [ ] ~~Admin Panel~~ - Not created yet

### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Wiki Updates

- [x] No need for updates
- [ ] Requires an update ( _not done within this issue_ )
- [ ] Has been updated ( _done within this issue_ )

### Testing checklist

- [ ] Unit tests
- [ ] Functional tests
- [ ] E2E tests

#### Compatibility tests - **_For E-Learning React App Only_**

- [ ] Tested on Safari
- [ ] Tested on Chrome
- [ ] Tested on Mozilla Firefox

### What changes have been done within this issue?

I have updated the pull request template to check if the assignees have added security filters for an endpoint.

### How should this be manually tested?

By reading the pull request template.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ Closes: #<!-- Insert issue number here -->
 - [ ] Functional tests
 - [ ] E2E tests
 
+#### Security Filters - **_For API Only_**
+
```
Comment (Vimpat) [inline .github/pull_request_template.md:30]: Test review

Comment (sebastianbadea197) [inline .github/pull_request_template.md:30]:
```suggestion
**_Please make sure to add the respective Security Filters before submitting a PR_**
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ Closes: #<!-- Insert issue number here -->
...
+- [ ] They have been added within this issue.
+- [ ] They need to be added.
```
Comment (sebastianbadea197) [inline .github/pull_request_template.md]:
```suggestion
```

**Review by sebastianbadea197 — CHANGES_REQUESTED** [review]

**Review by sebastianbadea197 — DISMISSED** [review]

**Review by sebastianbadea197 — APPROVED** [review]
LGTM!

**Pull request merged**